config/config.go

```diff
@@ -200,38 +197,6 @@ func (c *Config) GetDatabaseParams() dbparams.Database {
 	}
 }
 
-type AwsS3RetentionConfig struct {
```

Comment:
It seems your PR is mixed up with retention deletion commit.

Reply:
let me know if you still see it after the merge with master.